src/tests/components/Login.test.tsx

```diff
@@ -14,51 +15,105 @@ const renderComponent: ReactElement = (
     </ServiceProvider>
 )
 
-test('Login page render', () => {
-    render(renderComponent)
-    const loginPrimaryText = screen.getByText('Rucio Login')
-    expect(loginPrimaryText).toBeInTheDocument()
+const multi_vo = env('multi_vo_enabled')?.toLowerCase() === 'true'
 
-    const loginSecondaryText = screen.getByText('Welcome to Rucio!')
-    expect(loginSecondaryText).toBeInTheDocument()
+const itif = (condition: boolean) => (condition ? it : it.skip)
 
-    const x509PassAuthText = screen.getByText('x509 Certificate')
-    expect(x509PassAuthText).toBeInTheDocument()
+describe('Login page render', () => {
+    test('Titles', () => {
+        render(renderComponent)
+        const loginPrimaryText = screen.getByText('Rucio Login')
+        expect(loginPrimaryText).toBeInTheDocument()
 
-    const userPassAuthText = screen.getByText('Username / Password')
-    expect(userPassAuthText).toBeInTheDocument()
-})
+        const loginSecondaryText = screen.getByText('Welcome to Rucio!')
+        expect(loginSecondaryText).toBeInTheDocument()
```

Comment:
This check is redundant. If `loginSecondaryText` is not in the document, `screen.getByText('Welcome to Rucio!')` would raise an `Object.getElementError` error :) 
Shall we git rid of them? I'm open to ideas on what tests can be considered `valuable`

Reply:
Yes that makes sense to me